### DISCUSSION TITLE
Using notion-like template as React component

### DISCUSSION THREAD

**Discussion by Sinkler521** [ueberdosis/tiptap#5644] [General Questions]
Hello and thanks for the job you've done with tiptap.

I followed instructions and got 2 folders with templates (a something like classic editor and notion-like). But those are separated projects I can run via npm run dev after installing packages. 
My question is: d**o I have a possibility to use these templates like a React component? As simple as the other ones that just installed, imported and used as JSX?** If there is some way, please let me know. 
I'm talking about editor with no ai and no pro functions at the start, just implement in my project the tiptap editor as React component?

Thank you, waiting for your response
![templates](https://github.com/user-attachments/assets/9f0317a4-c1d2-4c0c-b2af-b27875a32f02)

**Comment by nperez0111** [comment]
The notion-like template is built as an entire application so there is not a way right now to have this just be a simple react component

**Comment by hamishll** [comment]
Do either of you have a link to the notion-like template repo? Would love to play around with it! @nperez0111 @Sinkler521

**Reply by nperez0111** [reply]
To access our block editor template, simply head over to your [Tiptap Account](https://cloud.tiptap.dev/react-templates) If you are not already a member, sign up for an account. Once finished, you'll get an invite to the private GitHub repository.

**Reply by hamishll** [reply]
Thanks Nick! I'll check that out

**Reply by davidmacfarlan** [reply]
> To access our block editor template, simply head over to your [Tiptap Account](https://cloud.tiptap.dev/react-templates) If you are not already a member, sign up for an account. Once finished, you'll get an invite to the private GitHub repository.

Hi Nick.
We have a Tiptap team plan.
Can you invite me to the Nootion private GitHub repository you mentioned?
